### PR TITLE
update softmax_with_cross_entropy_cn.rst, test=develop

### PR DESCRIPTION
**Pull request by JepsonWong**
update softmax_with_cross_entropy_cn.rst.

### CODE REVIEW

**Comment by JepsonWong** [comment]
最新截图如下：
softmax_with

File: doc/fluid/api_cn/layers_cn/softmax_with_cross_entropy_cn.rst [hunk 1]
```diff
@@ -5,47 +5,45 @@ softmax_with_cross_entropy
 
 .. py:function:: paddle.fluid.layers.softmax_with_cross_entropy(logits, label, soft_label=False, ignore_index=-100, numeric_stable_mode=True, return_softmax=False, axis=-1)
 
-使用softmax的交叉熵在输出层已被广泛使用。该函数计算输入张量在axis轴上的softmax标准化值，而后计算交叉熵。通过此种方式，可以得到更具数字稳定性的梯度值。
+该OP实现了带有交叉熵损失的softmax层，其在输出层已被广泛使用。该函数首先计算输入在 ``axis`` 维度上的softmax归一化值，之后计算交叉熵损失。这种计算方式提供了数值上更稳定的梯度值。
```
Comment (phlrain) [inline doc/fluid/api_cn/layers_cn/softmax_with_cross_entropy_cn.rst]: 该函数首先计算输入在 ``axis`` 维度上的softmax归一化值，之后计算交叉熵损失。
这句话其实和参数内部实现是不完全一致的，内部实现的 softmax 和 log 计算 做了fuse

File: doc/fluid/api_cn/layers_cn/softmax_with_cross_entropy_cn.rst [hunk 2]
```diff
@@ -5,47 +5,45 @@ softmax_with_cross_entropy
...
 
-因为该运算是在内部进行logit上的softmax运算，所以它需要未标准化（unscaled）的logit。该运算不应该对softmax运算的输出进行操作，否则会得出错误结果。
+因为该运算在内部对 ``logits`` 执行softmax运算，所以它需要未标准化的 ``logits`` 。该运算不应该对softmax运算的输出进行操作，否则会产生错误的结果。
```
Comment (phlrain) [inline doc/fluid/api_cn/layers_cn/softmax_with_cross_entropy_cn.rst]: 未标准化的 

这句话是什么意思

File: doc/fluid/api_cn/layers_cn/softmax_with_cross_entropy_cn.rst [hunk 3]
```diff
@@ -5,47 +5,45 @@ softmax_with_cross_entropy
...
 
-当 ``soft_label`` 为 ``False`` 时，该运算接受互斥的硬标签，batch中的每一个样本都以为1的概率分类到一个类别中，并且仅有一个标签。
+当 ``soft_label`` 为 ``False`` 时，该运算期望互斥的硬标签，批次中的每一个样本都以1.0的概率分类到一个类别中，其仅有一个标签。
```
Comment (phlrain) [inline doc/fluid/api_cn/layers_cn/softmax_with_cross_entropy_cn.rst]: 该运算期望互斥的硬标签，批次中的每一个样本都以1.0的概率分类到一个类别中，其仅有一个标签。

这句话还是很 很看懂

File: doc/fluid/api_cn/layers_cn/softmax_with_cross_entropy_cn.rst [hunk 4]
```diff
@@ -5,47 +5,45 @@ softmax_with_cross_entropy
...
-3.如果 ``numeric_stable_mode`` 为真，在通过softmax和标签计算交叉熵损失前， softmax 首先经由下式计算得出：
+3. 如果 ``numeric_stable_mode`` 为 ``True`` ，softmax结果首先经由下式计算得出，然后通过softmax结果和 ``label`` 计算交叉熵损失。
 
 .. math::
     max_j           &= \max_{i=0}^{K}{\text{logit}_i} \\
     log\_max\_sum_j &= \log\sum_{i=0}^{K}\exp(logit_i - max_j)\\
     softmax_j &= \exp(logit_j - max_j - {log\_max\_sum}_j)
 
+参数：
+  - **logits** (Variable) - 维度为任意维的多维 ``tensor`` ，数据类型为float32或float64。表示未标准化的输入。
```
Comment (phlrain) [inline doc/fluid/api_cn/layers_cn/softmax_with_cross_entropy_cn.rst]: 未标准化，还是 未 归一化

File: doc/fluid/api_cn/layers_cn/softmax_with_cross_entropy_cn.rst [hunk 5]
```diff
@@ -5,47 +5,45 @@ softmax_with_cross_entropy
...
+  - **label** (Variable) - 如果 ``soft_label`` 为True， ``label`` 是一个和 ``logits`` 维度相同的的 ``Tensor`` 。如果 ``soft_label`` 为False， ``label`` 是一个在axis维度上大小为1，其它维度上与 ``logits`` 维度相同的 ``Tensor`` 。
+  - **soft_label** (bool, 可选) - 指明是否将输入标签当作软标签。默认值：False。
```
Comment (phlrain) [inline doc/fluid/api_cn/layers_cn/softmax_with_cross_entropy_cn.rst:36]: 可选 不是在取值包含None的时候， soft_label 这个就是true false的选项吧

File: doc/fluid/api_cn/layers_cn/softmax_with_cross_entropy_cn.rst [hunk 6]
```diff
@@ -5,47 +5,45 @@ softmax_with_cross_entropy
...
-  - **return_softmax** (bool) – 标志位，指明是否额外返回一个softmax值， 同时返回交叉熵计算结果。默认为False。
-  - **axis** (int) – 执行softmax计算的维度索引。 它应该在范围 :math:`[ -  1，rank  -  1]` 中，而 :math:`rank` 是输入logits的秩。 默认值：-1。
-
-返回:
-  - 如果 ``return_softmax`` 为 False，则返回交叉熵损失
-  - 如果 ``return_softmax`` 为 True，则返回元组 (loss, softmax) ，其中softmax和输入logits形状相同；除了axis维上的形为1，其余维上交叉熵损失和输入logits形状相同
-
-返回类型:变量或者两个变量组成的元组
+返回：
+  - 如果 ``return_softmax`` 为 False，则返回交叉熵损失结果的 ``Tensor`` ，数据类型和 ``logits`` 一致，除了 ``axis`` 维度上的形状为1，其余维度和 ``logits`` 一致。
```
Comment (phlrain) [inline doc/fluid/api_cn/layers_cn/softmax_with_cross_entropy_cn.rst:43]: 现在softmax_with_cross_enttropy升级了吧，不强制要求最后一维等于1了

File: doc/fluid/api_cn/layers_cn/softmax_with_cross_entropy_cn.rst (hunk 1 above)
Comment (JepsonWong) [reply doc/fluid/api_cn/layers_cn/softmax_with_cross_entropy_cn.rst]: done

File: doc/fluid/api_cn/layers_cn/softmax_with_cross_entropy_cn.rst (hunk 2 above)
Comment (JepsonWong) [reply doc/fluid/api_cn/layers_cn/softmax_with_cross_entropy_cn.rst]: done

File: doc/fluid/api_cn/layers_cn/softmax_with_cross_entropy_cn.rst (hunk 3 above)
Comment (JepsonWong) [reply doc/fluid/api_cn/layers_cn/softmax_with_cross_entropy_cn.rst]: done

File: doc/fluid/api_cn/layers_cn/softmax_with_cross_entropy_cn.rst (hunk 4 above)
Comment (JepsonWong) [reply doc/fluid/api_cn/layers_cn/softmax_with_cross_entropy_cn.rst]: done

File: doc/fluid/api_cn/layers_cn/softmax_with_cross_entropy_cn.rst (hunk 6 above)
Comment (JepsonWong) [reply doc/fluid/api_cn/layers_cn/softmax_with_cross_entropy_cn.rst:43]: 目前还是要求最后一维为1

File: doc/fluid/api_cn/layers_cn/softmax_with_cross_entropy_cn.rst (hunk 5 above)
Comment (JepsonWong) [reply doc/fluid/api_cn/layers_cn/softmax_with_cross_entropy_cn.rst:36]: done

File: doc/fluid/api_cn/layers_cn/softmax_with_cross_entropy_cn.rst [hunk 7]
```diff
@@ -5,65 +5,54 @@ softmax_with_cross_entropy
 
 .. py:function:: paddle.fluid.layers.softmax_with_cross_entropy(logits, label, soft_label=False, ignore_index=-100, numeric_stable_mode=True, return_softmax=False, axis=-1)
 
-使用softmax的交叉熵在输出层已被广泛使用。该函数计算输入张量在axis轴上的softmax标准化值，而后计算交叉熵。通过此种方式，可以得到更具数字稳定性的梯度值。
+该OP实现了带有交叉熵损失的softmax层。该函数会将softmax操作、交叉熵损失函数的计算过程进行合并，从而提供了数值上更稳定的梯度值。
```
Comment (JiabinYang) [inline doc/fluid/api_cn/layers_cn/softmax_with_cross_entropy_cn.rst]: 交叉熵损失的softmax层，这个描述感觉不是很准确

**Review by JiabinYang — COMMENTED** [review]
a comment

File: doc/fluid/api_cn/layers_cn/softmax_with_cross_entropy_cn.rst (hunk 7 above)
Comment (JepsonWong) [reply doc/fluid/api_cn/layers_cn/softmax_with_cross_entropy_cn.rst]: done

**Review by JiabinYang — APPROVED** [review]
LGTM

File: doc/fluid/api_cn/layers_cn/softmax_with_cross_entropy_cn.rst (hunk 6 above)
Comment (lanxianghit) [reply doc/fluid/api_cn/layers_cn/softmax_with_cross_entropy_cn.rst:43]: 应该是不强制要求最后一维为1，请确认

Comment (JepsonWong) [reply doc/fluid/api_cn/layers_cn/softmax_with_cross_entropy_cn.rst:43]: 看了一下相关源码，并且咨询了一下OP负责人，这个OP目前还是要求最后一维为1。

**Review by lanxianghit — APPROVED** [review]
LGTM
